src/components/AddActivityPage/AddActivityPage.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import styled from 'styled-components';
 
 import AcivityForm from '../ActivityForm/ActivityForm';
 import { startAddActivity } from '../../store/activities/actions';
 import { ACTIVITY_PLAN_ROUTE } from '../../constants/routes';
 import historyPushPropTypeShape from '../../prop-types/history';
+import { H1 } from '../../styles/elements/H1';
+import { spacing } from '../../styles/base/base';
+
+const AddActivityWrapper = styled.div`
+  padding: ${spacing.sSize};
+
+  @media (min-width: ${spacing.desktopBreakpoint}) {
+    margin: 0 auto;
+    width: 80%;
+  }
+  @media (min-width: ${spacing.largeDesktopBreakpoint}) {
+    margin: 0 auto;
+    width: 40%;
+  }
+`;
 
 const AddActivityPage = ({ onStartAddActivity, history }) => (
-  <div>
-    <h1>Add Activity</h1>
+  <AddActivityWrapper>
+    <H1>Add Activity</H1>
```

Comment:
why this change?

Reply:
ahhh... I see.
It wouol be better to create a component `heading` which will receive prop, called `tag`. 
```
<Heading tag="h2"....>
```